lib/iris/tests/unit/fileformats/cf/test_CFReader.py

```diff
@@ -53,29 +49,31 @@ def netcdf_variable(
     return ncvar
 
 
-class Test_translate__global_attributes(tests.IrisTest):
-    def setUp(self):
+class Test_translate__global_attributes:
+    @pytest.fixture(autouse=True)
+    def _setup(self, mocker):
         ncvar = netcdf_variable("ncvar", "height", np.float64)
         ncattrs = mock.Mock(return_value=["dimensions"])
         getncattr = mock.Mock(return_value="something something_else")
-        self.dataset = mock.Mock(
+        dataset = mock.Mock(
```

Comment:
@pp-mo Is there a reason to use `mock` instead of `mocker` here?

Reply:
See above